ui/src/components/settings/SettingTable.js

```diff
@@ -355,6 +366,30 @@ class SettingTable extends React.Component {
                 />
             );
 
+        let auditEnabledDesc =
+            'Flag indicates whether or not ' +
+            this.props.category +
+            ' updates require explicit auditing approval process';
+        this.props.category === 'role' &&
+            this.props.justificationRequired &&
```

Comment:
justificationRequired there is no need to pass it from father component can take it from redux

Reply:
need to add || category === group

Reply:
removed justificationRequired and added the group category (apparently i didn't push the changes from vzm-ui, but now its synched.)